src/structs.jl

```diff
@@ -175,8 +176,8 @@ struct Model
     compartments::Dict{String,Compartment}
     species::Dict{String,Species}
     reactions::Dict{String,Reaction}
-    gene_products::Dict{String,GeneProduct}
-    function_definitions::Dict{String,FunctionDefinition}
+    gene_products::Maybe{Dict{String,GeneProduct}}
+    function_definitions::Maybe{Dict{String,FunctionDefinition}}
```

Comment:
Why have Maybe here? IMHO it may make some sense for the gene products (there might be a difference between "there are no gene products linked" vs "we don't talk about gene products here"), but certainly not for fundefs, if these are empty then they are just empty with no additional semantics right?

Reply:
Makes sense. Fixed with a1af25a. I also took Maybe out of gene_products, because I do not see how the difference between "there are no gene products linked" vs "we don't talk about gene products here" can be seen in the SBML file.

Reply:
The changes I made in a1af25a caused test cases to break. fixed in 44d88f5.